Composer/packages/lib/shared/src/featureFlagUtils/index.ts

```diff
@@ -6,10 +6,11 @@ import { FeatureFlagMap } from '@botframework-composer/types';
 
 export const getDefaultFeatureFlags = (): FeatureFlagMap => ({
   NEW_CREATION_FLOW: {
-    displayName: formatMessage('New Creation Experience'),
-    description: formatMessage('Preview the new adaptive runtime and component system.'),
+    displayName: formatMessage('The Component Model'),
+    description: formatMessage('Enable the Component Model including the new creation experience, the Adaptive Runtime, and Package Manager.'),
     isHidden: false,
     enabled: false,
+    documentationLink: 'https://aka.ms/AAbf42g',
```

Comment:
Why the obfuscated aka link?

Reply:
I feel like that is best practice, In case we want to change the location of the documentation post release we can do it through the aka portal. Not sure how likely that is but better have the ability to do then not.

Reply:
Most (if not all) of our other aka links use a friendly name (ex. https://aka.ms/bfcomposer).

Reply:
Ah I see, so use aka but a friendly name, gotcha, I have made the change and pushing latest iteration now